cmd/eksctl-anywhere/cmd/importimages.go

```diff
@@ -146,3 +148,7 @@ func getChartUriAndVersion(chart v1alpha1.Image) (uri, version string) {
 	version = chart.Tag()
 	return uri, version
 }
+
+func pushChartURI(chart v1alpha1.Image, registryEndpoint string) string {
+	return urls.ReplaceHost(ociPrefix+filepath.Dir(chart.Image()), registryEndpoint)
```

Comment:
nit: Sprintf instead of `+`
```suggestion
	orgURL := fmt.Sprintf("%s%s", ociPrefix, filepath.Dir(chart.Image()))
	return urls.ReplaceHost(orgURL, registryEndpoint)
```